guides/common/modules/proc_upgrading-your-project-server.adoc

```diff
@@ -18,6 +18,14 @@ include::snip_warning-maintain-config-noop.adoc[]
 ----
 +
 This prevents the upgrade process from overwriting your DNS and DHCP configuration.
+ifdef::upgrading-connected[]
+* If you use Insights in your environment, update to the latest container images:
```

Comment:
```suggestion
* If you use Insights in your environment, ensure you're logged in to `registry.redhat.io` so that the latest container images can be downloaded:
```